contracts/Vault.vy

```diff
@@ -741,7 +763,7 @@ def decreaseAllowance(spender: address, amount: uint256) -> bool:
 
 
 @external
-def permit(owner: address, spender: address, amount: uint256, expiry: uint256, signature: Bytes[65]) -> bool:
+def permit(owner: address, spender: address, amount: uint256, deadline: uint256, v: uint8, r: bytes32, s: bytes32) -> bool:
```

Comment:
looping in infra

Reply:
agree it makes sense to move to 0.4.5 to avoid confusion , so i guess next steps is release 0.4.4 and then 0.4.5 with this change

Reply:
As a hack, you can define the `permit` function to be 2612 compliant, and then farm another function name that has the same method ID as `permit(address,address,uint256,uint256,Bytes[65])` and accepts the same type arguments, and then just have that split the signature into components and forward to the correct method